### DISCUSSION TITLE
Site authentication settings webmaster_verifications: Inquiry regarding this

### DISCUSSION THREAD

**Discussion by LeeJuhyeong424** [cotes2020/jekyll-theme-chirpy#1988] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

Looking at the config file
There is a webmaster_verifications settings tab.
google: # Enter your Google verification code.
  bing: # Enter your Bing verification code.
  alexa: # Enter your Alexa authentication code.
  yandex: # Enter Yandex verification code.
  baidu: # Enter Baidu authentication code.
  facebook: # Enter your Facebook verification code.

I wonder where and how to load the code if I enter it. When I check f12 after building the website, I can see that the value is being loaded normally. What file are you dragging in to save it as that variable? Additionally, I would like to add a variable because I need an additional meta tag. (naver) When saving the variable for direct addition, can I know which file it is loaded from?

### Operations you have already tried

I checked all source files to find the tag, but I can't find it.

**Comment by kungfux** [comment]
You get the verification code from the search engine and put it in the config file. See e.g. [Google documentation](https://support.google.com/webmasters/answer/9008080?hl=en) and [HTML tag verification](https://support.google.com/webmasters/answer/9008080?hl=en#meta_tag_verification) specifically for more information.

If you need to add additional meta tags that are not supported by Chirpy, you can create a file `_includes/metadata-hook.html` and define it like follows:

```html
<meta name="name" content="value">
```
